src/apps/answers/tests/test_answers.py

```diff
@@ -3517,7 +3544,7 @@ async def test_validate_multiinformant_assessment_fail_no_applet(self, client, l
 
         response = await client.get(url)
 
-        assert response.status_code == http.HTTPStatus.NOT_FOUND
+        assert response.status_code == http.HTTPStatus.UNPROCESSABLE_ENTITY
 
```

Comment:
this test should continue returning NOT_FOUND, you should pass all parameters, but a wrong appleId

Reply:
Thanks Rodrigo. I think I missed a commit before pushing. Let me add the changes.